src/clr-angular/layout/vertical-nav/vertical-nav.html

```diff
@@ -6,6 +6,8 @@
 
 <button type="button" class="nav-trigger"
         [class.on-collapse]="collapsed"
+        [attr.aria-expanded]="ariaExpanded"
```

Comment:
Why is this `ariaExpanded` here and `expanded` in `vertical-nav-group.html`?

Reply:
The original issue was for vertical nav groups. The external contributor added this one (the one that controls the entire vertical nav) and I left these changes alone.

I tested VoiceOver with both expanding this one and for the vertical nav groups.